### PR TITLE
Update international workshop form

### PR DESCRIPTION
**Pull request by mgc1194**
The [international Workshop Attendance form](https://studio.code.org/pd/international_workshop?lang=en-US) is used to collect and track information about workshop attendees.
The international team has requested to do some changes to this form. 
The changes requested are:
- Remove the alternate email field.
- Remove the gender identity field.
- Remove the ages-you-teach question.
- Remove the subjects-you-teach question.
- Remove the CS resources question.
- Remove the Robotics resources question.
- Remove the workshop facilitator.
- Add new countries to the school-country dropdown.
- Add new partners to the partners' list.
- Update legal text

## Links

- spec: [IPD Update](https://docs.google.com/spreadsheets/d/1rYMWWgbYrSlepdLLz4uQNk9750tGtgOi/edit#gid=1530512169)
- Jira ticket: [P20-200](https://codedotorg.atlassian.net/browse/P20-200)

## Testing story
image

Updated Apps test and successfully passed:
```
npm run test:entry -- --entry=./test/unit/code-studio/pd/international_opt_in/InternationalOptInTest.jsx 


SUMMARY:
✔ 13 tests completed
Done.
```
Updated Dashboard test and successfully passed: 
```
 bundle exec spring testunit ./test/controllers/api/v1/pd/workshops_controller_test.rb

  111/111: [==========================================================================================================================================================] 100% Time: 00:00:30, Time: 00:00:30
Finished in 40.27125s
111 tests, 269 assertions, 0 failures, 0 errors, 0 skips
```
```
bundle exec spring testunit ./test/models/pd/international_opt_in_test.rb

  2/2: [==============================================================================================================================================================] 100% Time: 00:00:09, Time: 00:00:09
Finished in 11.12204s
2 tests, 7 assertions, 0 failures, 0 errors, 0 skips
```

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by wilkie — APPROVED** [review]
It is difficult to gauge the correctness of what you've removed. I don't have the context. But the changes seem consistent and make sense from the context I do have. Only one strange thing I noticed that is just me not understanding, perhaps. Otherwise, if you get / have further approval for the content, well, LGTM 👍 :)

File: dashboard/test/models/pd/international_opt_in_test.rb [hunk 1]
```diff
@@ -29,7 +22,7 @@ class Pd::InternationalOptInTest < ActiveSupport::TestCase
     refute build(:pd_international_opt_in, form_data: {}.to_json, user_id: teacher.id).valid?
 
     refute build(
-      :pd_international_opt_in, form_data: FORM_DATA.merge({ages: nil}).to_json, user_id: teacher.id
+      :pd_international_opt_in, form_data: FORM_DATA.merge({schoolName: nil}).to_json, user_id: teacher.id
```
Comment (wilkie) [inline dashboard/test/models/pd/international_opt_in_test.rb:25]: This is a change I don't quite understand. What changed exactly to assert something so different in this test? I'm sure its just smoothing out the content changes and the requested data, but it is a surprising change when not much changed around it in the test.

Comment (mgc1194) [reply dashboard/test/models/pd/international_opt_in_test.rb:25]: My understanding was that this test evaluates an invalid required input, to make sure it does not allow the form submission without that response. I arbitrarily picked a different required field after removing the ages field.

Comment (wilkie) [reply dashboard/test/models/pd/international_opt_in_test.rb:25]: That was my intuition and that makes sense to me!

**Review by wilkie — APPROVED** [review]

**Pull request merged**